src/graph_gui.cpp

```diff
@@ -555,20 +633,23 @@ struct BaseGraphWindow : Window {
 
 	void DrawWidget(const Rect &r, WidgetID widget) const override
 	{
-		if (widget != WID_GRAPH_GRAPH) return;
-
-		DrawGraph(r);
+		if (widget == WID_GRAPH_GRAPH) {
+			DrawGraph(r);
+		}
```

Comment:
Why change this? Early returns are generally nicer than nesting and this is functionally identical.